applications/actian-zen/README.md

```diff
@@ -0,0 +1,12 @@
+```text
+SPDX-License-Identifier: Apache-2.0
+Copyright (c) 2020 Actian Corporation
+```
+# Actian Zen 
+Actian Zen Edge Data Management is a single, secure, modular, and scalable solution that embeds in or bundles with applications for remote/branch, mobile, and IoT data processing and analytics support. Zen provides SQL and NoSQL access for any type of data from within most popular programming languages running on Windows (IoT Core, Desktop, Server), Linux (embedded and standard), MacOS running on Intel platforms and running on iOS/Android for mobile.
+
+For more information on Actian Zen go to https://www.actian.com/data-management/zen-embedded-database/
+
+## Install
+Helm charts are provided in the 'helm-charts' directory to assist with the deployment of the Zen database and sample application in an OpenNESS environment.
```

Comment:
Can you elaborate more on the purposes of the two helm charts provided: `actianzen` and `zensample` ?

Reply:
Additional info provided in latest commit.